backend/main.py

```diff
@@ -3,10 +3,10 @@
 from itertools import groupby
 from typing import Dict, List, Optional, Union
 
-from bson import ObjectId
-from fastapi import FastAPI, HTTPException
-from motor.motor_asyncio import AsyncIOMotorClient
-from pydantic import BaseModel
+from bson import ObjectId # type: ignore
+from fastapi import FastAPI, HTTPException # type: ignore
+from motor.motor_asyncio import AsyncIOMotorClient # type: ignore
+from pydantic import BaseModel # type: ignore
```

Comment:
These shouldn't be ignored.

Reply:
Do you have these installed locally also?